scripts/test/js_test.sh

```diff
@@ -51,6 +51,7 @@ if [[ $(
     grep -v "Accessing PropTypes" |
     grep -v "Accessing createClass" |
     grep -v "Browserslist: caniuse-lite is outdated" |
+    grep -v "npx browserslist@latest" |
```

Comment:
Why was this change necessary?

Reply:
Fixes test errors on github (https://github.com/mitodl/open-discussions/pull/3247)